CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @davidzchen
```

Comment:
David is no longer active. Last time, I failed to find any maintainer or user of rules_d.

Since I last fixed the rules, I'm fine with being the owner, until someone steps up.